frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -470,8 +470,12 @@ export const eventUsageLogic = kea<eventUsageLogicType>({
             dashboardId,
         }),
         reportInstanceSettingChange: (name: string, value: string | boolean | number) => ({ name, value }),
+        axisUnitsChanged: (properties: Record<string, any>) => ({ ...properties }),
```

Comment:
Should this not be "reportAxisUnitsChanged" ?

Reply:
I hate almost everything about this logic 🤣. It's a massive bag of stuff. 

But that name would at least be consistent